Week2task1/src/ComplexNumber.java

```diff
@@ -0,0 +1,79 @@
+
+public class ComplexNumber {
+	Integer real;
+	Integer im;
+	/**
+	 * @param real
+	 * @param im
+	 */
+	public ComplexNumber(Integer real, Integer im) {
+		super();
+		this.real = real;
+		this.im = im;
+	}
+
+	public ComplexNumber() {
+		// TODO Auto-generated constructor stub
+	
+	}
+	
+	public Integer getReal() {
+		return real;
+	}
+
+	public void setReal(Integer real) {
+		this.real = real;
+	}
+
+	public Integer getIm() {
+		return im;
+	}
+
+	public void setIm(Integer im) {
+		this.im = im;
+	}
+	// receiving as argument another ComplexNumber and doing the sum of the numbers
+	public static ComplexNumber sum(ComplexNumber num1, ComplexNumber num2) {
+		ComplexNumber sum = new ComplexNumber();
+		sum.real = num1.real + num2.real;
+		sum.im = num1.im + num2.im;
+		return sum;
+	}
+	// receiving as param another ComplexNumber and doing the product of the numbers
+	public static ComplexNumber product(ComplexNumber num1, ComplexNumber num2) {
+		ComplexNumber product = new ComplexNumber();
+		product.real = (num1.real*num2.real - num1.im*num2.im);
+		product.im = (num1.real*num2.im + num2.real*num1.im);
+		return product;
+	}
+	// printing the complex number in the format: a + bi (real + "+" + im + "*" + "i")
+	public String toString(ComplexNumber num1, ComplexNumber num2) {
+		String format = (real + "+" + im + "*" + "i");  
+		return format;
+	}
+	// testing if two complex numbers are equal
+	public static void equals(ComplexNumber num1, ComplexNumber num2) {
```

Comment:
I would make the equals method boolean and I would return true when the numbers are equal and false when they differ. The method equals from the parent Object is like this and is better that all the equals method we override return the same things as the equals from the parent.